.github/workflows/release.yml

```diff
@@ -15,6 +15,10 @@ env:
   SENTRY_ORG: hashicorp
   SENTRY_PROJECT: cdktf-cli
 
+concurrency:
+  group: release
+  cancel-in-progress: true
```

Comment:
Can't this be dangerous 😅
If something is merged while a release is running? Is there something we can do to safeguard that case other than "don't merge while we release a new version" 😄